src/databricks/labs/ucx/install.py

```diff
@@ -233,12 +234,36 @@ def warehouse_type(_):
             groups_config_args["selected"] = [x.strip() for x in selected_groups.split(",")]
         else:
             groups_config_args["auto"] = True
+
+        instance_profile = None
```

Comment:
i think the best UX for this is a drop-down for AWS Glue-enabled clusters and keep the `aws_glue_cluster_id` as the private field of the installer. and then create special job tasks in the assessment to crawl for tables and tacls from AWS Glue metastore.

Reply:
We don't need a special job. The existing one works with Glue (I confirmed it).